AEPAssurance.xcodeproj/project.pbxproj

```diff
@@ -941,6 +944,15 @@
 			knownRegions = (
 				en,
 				Base,
+				de,
+				es,
+				fr,
+				it,
+				ja,
+				ko,
+				"pt-BR",
```

Comment:
Do the other entries also need the country name? `de-De` .. ?

Reply:
No clue why they are different. I selected what was available here: 

![image](https://github.com/adobe/aepsdk-assurance-ios/assets/10753118/fe4f0807-96a5-464c-a463-c18e54466ed3)

Reply:
Oh, i think it only specifies country when there is a 1:N mapping of the language to different countries. Looks like Xcode doesn't have any other options for German.

Reply:
I see! Got it. 👍🏼

Reply:
NVM. Just found the de-DE. It was hidden. Fixing it now.